### ISSUE TITLE
Link to tools page from Setup and IDE Alternative Guide

### ISSUE THREAD

**Issue by ahoppen** [swiftlang/swift-org-website#745]
I just stumbled across the [Setup and IDE Alternative Guide](https://www.swift.org/documentation/server/guides/setup-and-ide-alternatives.html) and noticed that it has its own list of editors and how to set them up

Now that we have guides on how to set up editors at https://www.swift.org/tools/, I’m wondering if we still need that guide. I thought about linking to the tools page from the guide but that would leave the guide page pretty empty, so I think it might be better to adjust all pages that link to the Setup and IDE Alternatives guide to instead to the tools page.

**Comment by alexandersandberg** [comment]
@0xTim

**Comment by 0xTim** [comment]
Yeah we discussed this in the SSWG recently and it's something we'd rather hand over to a tools/platforms group so very happy to remove it
